finish/src/main/java/io/openliberty/guides/inventory/InventoryConfig.java

```diff
@@ -21,16 +21,16 @@
 @RequestScoped
 public class InventoryConfig {
 
-    @Inject
-    @ConfigProperty(name = "io_openliberty_guides_inventory_inMaintenance")
-    private Provider<Boolean> inMaintenance;
+	@Inject
+	@ConfigProperty(name = "io_openliberty_guides_inventory_inMaintenance")
+	private Provider<Boolean> inMaintenance;
 
-    public boolean isInvInMaintenance() {
-        try {
-            Thread.sleep(3000);
-        } catch (Exception e) {
-            e.printStackTrace();
-        }
-        return inMaintenance.get();
-    }
+	public boolean isInvInMaintenance() {
```

Comment:
Looks like there's indentation issue here.. pls use what's suggested by the guideline.